package.json

```diff
@@ -25,6 +25,7 @@
     "android:studio": "yarn workspace @my-app/mobile studio",
     "ios:metro": "yarn workspace @my-app/mobile start",
     "ios:start": "yarn workspace @my-app/mobile ios",
+    "ipad:start": "yarn workspace @my-app/mobile ipad",
```

Comment:
Can we avoid adding this script? You can run it anyway passing the simulator options to `yarn ios` 👍

Reply:
sorry about that one, I was using it on my project but somehow I made a pull request for that change, I submitted a new commit without it